movies/src/components/addMovie/AddMovie.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Input from '../common/input/Input';
+import Button from '../common/button/Button';
+import MultiSelectInput from '../common/multiSelectInput/MultiSelectInput';
+import Datepicker from '../common/datepicker/Datepicker';
+import CloseButton from '../closeButton/CloseButton';
+import MovieActionPopup from '../movieActionPopup/MovieActionPopup';
+import Constants from '../constants';
+
+export default function AddMovie(props) {
+  const [isMovieAction, isMovieActionPopup] = useState(false);
+  const { handleClose } = props;
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    isMovieActionPopup(true);
+  };
+
+  return (
+    <>
+      <CloseButton handleClose={handleClose} />
+      <form onSubmit={handleSubmit}>
+        <p className="modal-title">Add Movie</p>
+        <Input
+          id="movie-title"
+          title="Title"
+          labelClassName="input-label"
+        />
+        <Datepicker
+          id="movie-release"
+          title="Release date"
+          labelClassName="input-label"
+          value={this}
+        />
+        <Input
+          id="movie-url"
+          title="Movie url"
+          labelClassName="input-label"
+        />
+        <div className="modal-dropdown">
+          <span className="modal-label-text">Genre</span>
+          <MultiSelectInput options={Constants.GENRE} />
```

Comment:
![image](https://user-images.githubusercontent.com/16895020/93580132-0b710680-f9a8-11ea-8ccc-f74063942a29.png)
for me multiselect looks like this

Reply:
Styles for multySelect has been added